### PR TITLE
[CDAP-7951] Only current application is highlighted in product drawer

### PR DESCRIPTION
**Pull request by PGuimarais**
- Now in the product drawer, only the text of the active item appears as white

screen shot 2016-12-20 at 12 26 11 pm

JIRA:
https://issues.cask.co/browse/CDAP-7951

### CODE REVIEW

File: cdap-ui/app/cdap/components/ProductsDrawer/index.js [hunk 1]
```diff
@@ -105,10 +105,11 @@ export default class ProductsDrawer extends Component {
   }
   render() {
 
+    let topItemClasses = classnames("top-product", this.state.products[0].name, {'open' : this.state.productsDropdown, 'closed' : !this.state.productsDropdown, 'current-selection' : this.state.products[0].name === this.props.currentChoice});
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/ProductsDrawer/index.js]: ~~Lets not nest ternary operators. Its right now not very readable.~~
Its not ternary operator at all but lack of indentation.

**Comment by ajainarayanan** [comment]
Aside from the minor comment 👍 LGTM

File: cdap-ui/app/cdap/components/ProductsDrawer/ProductsDropdown.less [hunk 2]
```diff
@@ -37,126 +37,139 @@
...
+        border: 0;
+        height: ~"calc(100vh - 50px)";
+        height: ~"-webkit-calc(100vh - 50px)";
+        height: ~"-moz-calc(100vh - 50px)";
+        display: block;
+        transition: 0.1s ease-in;
+        left: 100%;
+        outline: 0;
+        user-select: none;
+        -moz-user-select: none;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/ProductsDrawer/ProductsDropdown.less]: no need for vendor prefix. Autoprefixer is in webpack now

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**